spec/steps/execute.fmf

```diff
@@ -13,6 +13,26 @@ description: |
     restraint
         use the restraint harness to execute beakerlib tests
 
+    For each ``plan`` the execute step should produce a ``results.yaml`` file
+    with the list of results for each test in the following format::
+
+        /test/one:
+            result: TEST_RESULT
+            log: PATH
+
+        /test/two:
+            result: TEST_RESULT
+            log: PATH
+
+    Where ``TEST_RESULT`` is the outcome of the test. It can have these values:
+
+        * info - test finished and produced only information message
+        * passed - test finished and passed
+        * failed - test finished and failed
+        * error - test did not finish, treated as failed
```

Comment:
Test can finish and still errors can happen during execution. What about `a problem encountered during test execution, treated as failed`?